README.md

```diff
@@ -404,10 +404,7 @@ If you are using ProGuard, we suggest disabling shrinking and obfuscation of ADA
 To disable obfuscation of ADAL classes:
 
 ```
--keep class com.microsoft.aad.adal.** {
-  *;
-}
--keep class com.microsoft.identity.common.** {
+-keep class com.microsoft.** {
```

Comment:
There's some more lines in this file I would expect to see changed
- Current version designator
- Minimum recommended version (probably 1.16.1 at this point)
- [This](https://github.com/AzureAD/azure-activedirectory-library-for-android#option-2-binaries-via-gradle)

Reply:
Should update minimum version to 1.16.3-hf2?

Reply:
Can't remember what the last crit fix for us was, I'll defer to your better judgement 😄